metron-deployment/packaging/ambari/src/main/resources/common-services/INDEXING/0.2.0BETA/service_advisor.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env ambari-python-wrap
+"""
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import imp
+import os
+import traceback
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+STACKS_DIR = os.path.join(SCRIPT_DIR, '../../../stacks/')
+PARENT_FILE = os.path.join(STACKS_DIR, 'service_advisor.py')
+
+try:
+    with open(PARENT_FILE, 'rb') as fp:
+        service_advisor = imp.load_module('service_advisor', fp, PARENT_FILE, ('.py', 'rb', imp.PY_SOURCE))
+except Exception as e:
+    traceback.print_exc()
+    print("Failed to load parent service_advisor file '{}'".format(PARENT_FILE))
+
+
+class INDEXING020BETAServiceAdvisor(service_advisor.ServiceAdvisor):
```

Comment:
If the Service Advisor can be dropped and doesn't affect the core functioning of your PR, then it would be nice to take it out. Less code is more.

It sounds like you have a go-forward plan to resolve the 'version embedded in the path' issue with Ambari mpacks.  I'm good with that plan.